storybook/stories/Layout/ModalWithClose.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+import { storiesOf } from "@storybook/react-native";
+import { ModalWithClose } from "../../../src/components/Layout/ModalWithClose";
+import { AppText } from "../../../src/components/Layout/AppText";
+
+import { View } from "react-native";
+
+const ModalItems = (): JSX.Element => {
```

Comment:
Will be better to put a button in the story itself so that it can be pressed to open the modal.

Reply:
added button to show the modal